### PR TITLE
🔨 Refactor, 🧠 Overmind, Hacktoberfest - /app/pages/common/Modals/PRModal/index.js: convert to tsx and uses overmind

### PR DESCRIPTION
**Pull request by ftonato**
## What kind of change does this PR introduce?

Refactors code as a part of hacktoberfest mentioned in #2621.
@Saeris @christianalfoni

## What is the current behavior?

/app/pages/common/Modals/PRModal/index.js was using inject from app/componentConnectors and written in JS.

## What is the new behavior?

uses useOvermind hook from app/overmind and converted to tsx.

## Checklist
* [ ]  Documentation
* [ ]  Testing
* [x]  Ready to be merged
* [ ]  Added myself to contributors table

### CODE REVIEW

**Comment by ftonato** [comment]
@lbogdan should I solve the conflict or you'll fix it?

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2803/commits/5c41c3b25e91b93703698c1b0e042f8e3a783e3e is at https://pr2803.build.csb.dev/s/new.

**Comment by ftonato** [comment]
@lbogdan is everything okay with this _pr_?

File: packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx [hunk 1]
```diff
@@ -4,8 +4,7 @@ import { useOvermind } from 'app/overmind';
 import React, { FunctionComponent } from 'react';
 import { Explanation, Heading } from '../elements';
 import { Container } from './elements';
-
-const LiveModeEnded: FunctionComponent = () => {
+export const LiveModeEnded: FunctionComponent = () => {
```
Comment (christianalfoni) [inline packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx]:
```suggestion

export const LiveModeEnded: FunctionComponent = () => {
```

Comment (christianalfoni) [reply packages/app/src/app/pages/common/Modals/LiveSessionEnded/index.tsx]: You are getting a linting error on this :)

**Comment by christianalfoni** [comment]
Added PR for you if you are unsure how to fix the import, it also includes the linting fix: https://github.com/ftonato/codesandbox-client/pull/1

You can just merge that and we should be good.

**Comment by ftonato** [comment]
@christianalfoni i did, tks o/

**Comment by ftonato** [comment]
Hello @christianalfoni, I have accepted your suggestion, can we merge now?

**Comment by CompuIves** [comment]
Ready for merge! Thanks!

**Pull request merged**